assets/src/app.js

```diff
@@ -28,8 +28,14 @@ function App() {
   const [authorFields, setAuthorFields] = useState({});
   // TODO: Add these.
   // const [resource, setResource] = useState(null);
+  const [resource, setResource] = useState(null);
+  const [facilities, setFacilities] = useState(null);
   // const [minimumSeatsRequired, setMinimumSeatsRequired] = useState(null);
 
+  // ResourceView overlay trigger & global data
+  const [showResourceView, setShowResourceView] = useState(null);
```

Comment:
Can these two states (showResourceView , resourceId) be merged to 

```
const [showResourceViewId, setShowResourceViewId] = useState(null);
```

and then you can deduct from whether or not showResourceViewId is null, whether or not you should show the ResourceView